oper8/rollout_manager.py

```diff
@@ -295,9 +307,11 @@ def rollout(  # pylint: disable=too-many-locals,too-many-statements
         phase1_failed = deploy_completion_state.failed()
         log.debug(
             "[Phase 1] Deploy result: %s",
-            "SUCCESS"
-            if phase1_complete
-            else ("FAILED" if phase1_failed else "INCOMPLETE"),
+            (
```

Comment:
Interesting. I'm guessing this was just a `black` reformat? Strange for it to get picked up like this, but 🤷

Reply:
I also spent alot of time puzzling over this, I also don't know why `black` reformatted it now

Reply:
I am using `black` formatter via VSCode (`Format Document With ...` > `Black formatter`), and that changed those lines. I also run `tox -e fmt,lint`, and all of the formatting and linting passed.